import shows

When attempting to import under foundry 0.6.4, I see:

` foundry.js:3826 Error: An error occurred while rendering AdventureModuleImport 35: This is not a registered game setting
    at ClientSettings.get (foundry.js:3023) `

adventure-import.js shows:
`    const importpath = game.settings.get("adventureimportexport", "importpath");`

when using the debugger, it looks like it is registered under adventure-import-export.import path.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,4 @@
+* 2020/06/29 - Bug fix for Module Configuration Option for Import Path
 * 2020/06/29 - Bug fix for Module Configuration Options
 * 2020/06/29 - Bug fix for import where game system has included compendiums, importer was aggressively trying to repair references.
 * 2020/06/29 - Enhancement - Displays the modules that were active in dialog upon import completion.

--- a/module.json
+++ b/module.json
@@ -2,7 +2,7 @@
   "name": "adventure-import-export",
   "title": "Adventure Importer / Exporter",
   "description": "Allows import and export of adventures (including all Foundy VTT Assets).",
-  "version": "0.2.5",
+  "version": "0.2.6",
   "author": "Chris Stadther",
   "minimumCoreVersion": "0.6.0",
   "compatibleCoreVersion": "0.6.4",
@@ -27,6 +27,6 @@
   "packs": [],
   "url": "https://github.com/cstadther/adventure-import-export",
   "manifest": "https://raw.githubusercontent.com/cstadther/adventure-import-export/master/module.json",
-  "download": "https://github.com/cstadther/adventure-import-export/archive/0.2.5.zip",
+  "download": "https://github.com/cstadther/adventure-import-export/archive/0.2.6.zip",
   "license" : "LICENSE"
 }

--- a/src/adventure-import.js
+++ b/src/adventure-import.js
@@ -11,7 +11,7 @@ class AdventureModuleImport extends FormApplication {
 
   /** @override */
   async getData() {
-    const importpath = game.settings.get("adventureimportexport", "importpath");
+    const importpath = game.settings.get("adventure-import-export", "importpath");
     let data;
     let files = [];
 

